ddl.dic

```diff
@@ -1154,6 +1176,73 @@ save_enumeration_default.value
 
 save_
 
+save_ENUMERATION_DEFAULTS
+
+    _definition.id                ENUMERATION_DEFAULTS
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-06-01
+    _description.text
+;
+    Loop of pre-determined default enumeration values indexed by a set of data
+    items specified using the _enumeration.def_index_ids attribute.
+;
+    _name.category_id             ENUMERATION
+    _name.object_id               ENUMERATION_DEFAULTS
+    _category_key.name            '_enumeration_defaults.index'
+
+save_
+
+save_enumeration_defaults.index
+
+    _definition.id                '_enumeration_defaults.index'
+    _definition.class             Attribute
+    _definition.update            2023-06-10
+    _description.text
+;
+    An index value, in a form of an ordered list, which allows the
+    identification the most suitable default value. The order of values in the
+    list must correspond to the order of the related data item references in the
+    _enumeration.def_index_ids attribute list. That is, the n^th^ list element
+    is assumed to represent a value that may be assigned to the data item
+    referenced at the n^th^ position of the _enumeration.def_index_ids attribute
+    list value.
+
+    The constituent values of the _enumeration_defaults.index attribute inherit
+    the enumeration range, enumeration set, and the content type of the
+    data item from which they are paired.
```

Comment:
```suggestion
    the enumeration range, enumeration set, and the content type from the
    data item with which they are paired.
```

Reply:
The "from which they are paired" part sounded a bit weird.

Reply:
Your suggestion is good.